### ISSUE TITLE
Should packet counts reset when a new file is loaded?

### ISSUE THREAD

**Issue by Bokonon79** [amund7/CANBUS-Analyzer#19]
I've noticed that if you load one log file, and then load a second file, the packets from the second file are added to the first file. 

This could be useful if someone wanted to analyze CAN messages from multiple capture sessions (e.g. beginning of a charging session + end of a charging session), but I'm not sure how common of a use-case that is. 

The other potential "gotcha" is that, if we start plotting the graphs based on timestamps (rather than sequence), multiple files might produce charts with large gaps in the middle.

Would it make more sense to clear out all of the packet counts when a new file is loaded?

**Comment by amund7** [comment]
Yes, I agree, it should start from zero with any new log file, feel free to implement if you see a quick way to do it.
